TestAssets/TestProjects/TestAppSimple/project.json

```diff
@@ -0,0 +1,12 @@
+{
+  "version": "1.0.0-*",
+  "compilationOptions": {
+    "emitEntryPoint": true
+  },
+  "dependencies": {
+    "Microsoft.NETCore.App": "1.0.0-rc2-*"
```

Comment:
Correct - you're adding another dependency on `--infer-runtimes` here, which is supposedly going away at some point...